machine/ingrasys/ingrasys_s8900_64xc/kernel/0002-driver-pca954x-i2c-mux-force-deselect-on-exit-flag.patch

```diff
@@ -0,0 +1,25 @@
+From 9f78c821edf12c8c91c290f591cd5a0681853592 Mon Sep 17 00:00:00 2001
```

Comment:
Please do not include the same patch, over and over for each machine.

You can share patches across machines by putting them in a top level machine/<vendor>/kernel directory.

For example see d43c000 which adds a common kernel patch used by multiple accton machine.

This makes it easier for you to maintain your machines and patches.